<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDevicesConfig</class>
 <widget class="QDialog" name="addDevicesConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add device</string>
  </property>
  <widget class="QPushButton" name="addDevConf_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>add device</string>
   </property>
  </widget>
  <widget class="QLabel" name="type_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>37</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="descr_lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="devTypes_comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="description_label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>description:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delDev_Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>delete device</string>
   </property>
  </widget>
  <widget class="QTableView" name="devices_tableView">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>431</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="devices_cBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
